libs/api-mocks/msw/handlers.ts

```diff
@@ -223,6 +224,61 @@ export const handlers = makeHandlers({
 
     return 204
   },
+  floatingIpCreate({ body, query }) {
+    const project = lookup.project(query)
+    errIfExists(db.vpcs, { name: body.name })
+
+    const newFloatingIp: Json<Api.FloatingIp> = {
+      id: uuid(),
+      project_id: project.id,
+      ip: body.address || '', // 👀 needs a legit ip
```

Comment:
This is necessarily going to be fakey without a robust mock IP situation, which I don't think is necessary yet. I think you should just stick in a dummy IP like `12.34.56.7`. We'll get duplicates if you create more than one, but I think that's ok for now.